distsql/select_result.go

```diff
@@ -448,10 +451,12 @@ func (s *selectResultRuntimeStats) String() string {
 			buf.WriteString(", rpc_num: ")
 			buf.WriteString(strconv.FormatInt(copRPC.Count, 10))
 			buf.WriteString(", rpc_time: ")
-			buf.WriteString(time.Duration(copRPC.Consume).String())
+			buf.WriteString(execdetails.FormatDuration(time.Duration(copRPC.Consume)))
+		}
+		if config.GetGlobalConfig().TiKVClient.CoprCache.Enable {
```

Comment:
Should we explicitly tell the user if the CoprCache is disabed?

Reply:
@breeswish PTAL

Reply:
I don't have strong opinion over this.